poleno/utils/test.py

```diff
@@ -67,6 +68,16 @@ def mock_func(*args, **kwargs):
         except MockException:
             pass
 
+@contextlib.contextmanager
+def reload_for_context(manager, module):
+    module = importlib.import_module(module)
```

Comment:
@martinmacko47 Aby to bolo univerzalne, mozeme to napisat ako `module = importlib.import_module(module) if isinstance(module, basestring) else module`

Reply:
IMHO nateraz staci nechat iba string